packages/amplify-provider-awscloudformation/src/setup-new-user.js

```diff
@@ -43,9 +44,13 @@ async function run(context) {
     },
   ]);
 
-  const deepLinkURL = constants.AWSCreateIAMUsersUrl.replace('{userName}', userName).replace('{region}', awsConfig.region);
+  let deepLinkURL = constants.AWSCreateIAMUsersUrl.replace('{userName}', userName).replace('{region}', awsConfig.region);
+  const isOnWindows = /^win/.test(process.platform);
```

Comment:
It's safe to use `process.platform === 'win32'` for these checks.

Reply:
Done. Thanks.